Index.html

```diff
@@ -33,6 +34,154 @@ <h2 class="subtitulo">I&apos;am a Software Developer</h2>
         </ul>
         </div>
     </section>
+
+    <section class="page1">
+      <h1 class="tituloMR">My Recent Works</h1>
+      <img class="vector"src="./images/Vector.svg">
+      <img class="yoga"src="./images/Img Placeholder.svg">
+      <h2 class="multi">Multi-Post Stories</h2>
+      <p class="parrafo1">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+      <div class="list_programs">
+        <ul>
+          <li<a><img class="css"src="./images/css.svg"></a></li>
+          <li<a><img class="htm"src="./images/html.svg"></a></li> 
+          <li<a><img class="bot"src="./images/boot.svg"></a></li>
+          <li<a><img class="ruby"src="./images/ruby.svg"></a></li>
+        </ul>
+      </div>
+      <button><a class="boton1">See Project</a></button> 
+
+      <div class="projects">
+        <h1 class="tituloPAPD">Profesional Art Printing Data</h1>
+        <p class="parrafo2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <div class="list_programs2">
+          <ul>
+            <li<a><img class="htm"src="./images/html2.svg"></a></li> 
+            <li<a><img class="bot"src="./images/boot2.svg"></a></li>
+            <li<a><img class="ruby"src="./images/ruby2.svg"></a></li>
+          </ul>
+        </div>
+          <button><a class="boton2">See Project</a></button> 
+        </div>
```

Comment:
- [ ] Your buttons appear to look different from the design template, kindly style them to look like the design template please as you are expected to stick to the design as much as possible.
- [Optional] Also, I will suggest you either use a `button` element or an `a` tag for your button, don't use both as you did.

- [ ] Your project cards appear to be aligned to the left, kindly fix that please and align them to the center. You are expected to stick to the design template as much as possible.

### Your design
![1](https://user-images.githubusercontent.com/94551484/198817476-e4a22d78-d53e-4a7d-a62e-87ef0dd45833.png)

### Figma template
![f1](https://user-images.githubusercontent.com/94551484/198817482-23e7471c-b6cf-4288-af06-89fb4bc00fd1.png)